### PR TITLE
Update cli

### PR DESCRIPTION
**Pull request by albaguirre**

### CODE REVIEW

**Comment by albaguirre** [comment]
@Saviq Are you ok with the wording for mount and find?

File: src/client/cmd/find.cpp [hunk 1]
```diff
@@ -118,13 +118,18 @@ QString cmd::Find::short_help() const
...
 
 mp::ParseCode cmd::Find::parse_args(mp::ArgParser* parser)
 {
-    parser->addPositionalArgument("string", "Optionally find images matching this string", "[<remote:>]<string>");
+    parser->addPositionalArgument("string",
+                                  "The value to search for in [<type:>]<string> format, where "
+                                  "<type> can be either ‘release’ or ‘daily’. If <type> is omitted, "
+                                  "it defaults to ‘release’. <string> can be a partial image hash or an "
+                                  "ubuntu release version, codename or alias",
+                                  "[<string>]");
```
Comment (Saviq) [inline src/client/cmd/find.cpp]: I think we need to do better to explain that you don't have to provide `<string>`.

Comment (albaguirre) [reply src/client/cmd/find.cpp]: @Saviq, what about "An optional value to search for in ...."

Comment (Saviq) [reply src/client/cmd/find.cpp]: +1

Comment (albaguirre) [reply src/client/cmd/find.cpp]: @Saviq ok fixed.

**Pull request merged**